tests/test_sklearn_kneighbors.py

```diff
@@ -27,13 +27,12 @@ def _test_kneighbors_classifier(
             X = X.astype(np.float32)
 
             if metric == "wminkowski":
-                metric_params["w"] = np.random.rand(X.shape[1])
+                metric_params = {"w": np.random.rand(X.shape[1])}
             elif metric == "seuclidean":
-                metric_params["V"] = np.random.rand(X.shape[1])
+                metric_params = {"V": np.random.rand(X.shape[1])}
             elif metric == "mahalanobis":
                 V = np.cov(X.T)
-                metric_params["V"] = V
-                metric_params["VI"] = np.linalg.inv(V)
```

Comment:
I couldn't seem to get this to work with both parameters, nor find an example with the new syntax. @scnakandala do you see an issue with just using the one param?

Reply:
![image](https://user-images.githubusercontent.com/1534199/145106742-2c0b9d89-3125-4e4b-9831-2813aef8d2f0.png)

So I think it is ok to just use VI